sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveExternalCatalog.scala

```diff
@@ -1309,6 +1312,8 @@ object HiveExternalCatalog {
 
   val CREATED_SPARK_VERSION = SPARK_SQL_PREFIX + "create.version"
 
+  val HIVE_GENERATED_STORAGE_PROPERTIES = Set(SERIALIZATION_FORMAT)
```

Comment:
@ueshin . The title means `Hive-generated storage properties`, but this PR excludes only this one. Could you add more? Othewise, can we make this as a SQLConf in order to be configurable?

Reply:
Actually the hive-generated storage property I think we should exclude for now is only this one, but we might have some more in the future, so I'd say "properties" and we will add them to this set in the case. WDYT?

Reply:
We can add more in the future. Basically, these properties are useless to Spark data source tables.

Reply:
I got it, @ueshin and @gatorsmile .